File: Bronze/Bronze_4/C_files/Code/Konpeito_22015.c
#include <stdio.h>

int main()
{
    int arr[3], max = 0, i;

    scanf("%d %d %d", &arr[0], &arr[1], &arr[2]);

    for (i = 0; i < 3; i++) {
        if (max < arr[i])
            max = arr[i];
    }

    printf("%d", max * 3 - (arr[0] + arr[1] + arr[2]));

    return 0;
}

File: Bronze/Bronze_4/Java_files/Code/Konpeito_22015.java
import java.util.Scanner;

public class Konpeito_22015 {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int arr[] = new int[3], max = 0;
        for (int i = 0; i < 3; i++) {
            arr[i] = sc.nextInt();
            if (max < arr[i])
                max = arr[i];
        }
        System.out.println(max * 3 - (arr[0] + arr[1] + arr[2]));
    }
}
